openreview/venue/process/review_process.py

```diff
@@ -8,20 +8,28 @@ def process(client, edit, invitation):
     authors_name = domain.get_content_value('authors_name')
     submission_name = domain.get_content_value('submission_name')
     reviewers_name = domain.get_content_value('reviewers_name')
+    pretty_reviewers_name = reviewers_name.replace('_', ' ')
     area_chairs_name = domain.get_content_value('area_chairs_name')
     senior_area_chairs_name = domain.get_content_value('senior_area_chairs_name')
     reviewers_submitted_name = domain.get_content_value('reviewers_submitted_name')
     review_name = domain.get_content_value('review_name')
     sender = domain.get_content_value('message_sender')
 
+    parent_invitation = client.get_invitation(invitation.invitations[0])
```

Comment:
I've added more checks here. First I check if the parent invitation has the booleans. If yes, use those. Otherwise, search users_to_notify. Once we move fully to the new venue config, we can get rid of the booleans